scripts/trigger_convergence.py

```diff
@@ -7,12 +7,18 @@
 Examples:
 `python trigger_convergence -c config.json -g "tenantid:groupId"`
 will trigger convergence on given group(s)
-`python trigger_convergence -c config.json`
+`python trigger_convergence -c config.json --all`
```

Comment:
Yes. Sorry this is correct example after implementing `--all`. Didn't change it then :)